gradle/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Build a JAR of updated Android models for a schema.
+# Takes three command-line arguments:
+# 1 - UUID of schema
+# 2 - the schema, as a string
+
+# use temp file for output jar file and temp dir for input schemas
+MODEL_JAR=`mktemp data/models.XXXXX.jar`
+SCHEMA_SOURCE_DIR=`mktemp -d data/schemas.XXXXX`
+ANDROID_KEYSTORE="data/driver.keystore"
+KEYSTORE_ALIAS="driver"
+TSA_URL="http://timestamp.digicert.com"
+# name of json file gets used by jsonschema2pojo for the base class
+DRIVER_SCHEMA_FILE="DriverSchema.json"
+
+# clean up temporary files on exit
+trap "echo 'Cleaning up...'; rm $MODEL_JAR; rm -rf $SCHEMA_SOURCE_DIR; echo 'All done!'" EXIT
+
+set -e
+
+# delete redis key for schema, if it exists
+redis-cli -h $DRIVER_REDIS_HOST -p $DRIVER_REDIS_PORT -n $DRIVER_REDIS_JAR_DB DEL $1
```

Comment:
Thinking about this more, I'm wondering if it's possible for certain access patterns on the endpoint to cause an endless delete-recompile loop here:

1. Client 1 requests Key A, not found -> Kicks off Job 1
2. Client 2 requests Key A, not found -> Kicks off Job 2
3. Job 1 Completes building Jar file, sets Redis key
4. Job 2 starts processing, deletes Redis key
5. Client 1 requests key A, not found -> kicks off Job 3
6. Job 2 finishes
7. Job 3 deletes Redis key
8. Goto 5